web/admin/application/configs/klear/model/CallCsvSchedulers.yaml

```diff
@@ -116,6 +138,73 @@ production:
             - name
             template: '%name%'
         'null': _("Use generic template")
+    ddi:
+      title: _('Outgoing DDI')
+      type: select
+      source:
+        data: mapper
+        'null': _("Any")
+        config:
+          entity: \Ivoz\Provider\Domain\Model\Ddi\Ddi
```

Comment:
Doesn't this require a filterClass?

Reply:
I've implemented it on the autocomplete command https://github.com/irontec/ivozprovider/pull/1320/commits/a7154fc932f71a00a3822a039a1ff76790928256#diff-2ec39302762541e4dda00583f8eafc9eR151

It filters both by brand or company depending on the screen: https://github.com/irontec/ivozprovider/pull/1320/commits/a7154fc932f71a00a3822a039a1ff76790928256#diff-08684761a5879e3ecf743dd999cbd149R27

Reply:
Cool, good job :+1: